src/OpenTK/Platform/SDL2/Sdl2NativeWindow.cs

```diff
@@ -424,19 +425,12 @@ private void DestroyWindow()
 
         private void GrabCursor(bool grab)
         {
-            SDL.ShowCursor(!grab);
             SDL.SetWindowGrab(window.Handle, grab);
-            SDL.SetRelativeMouseMode(grab);
-            if (!grab)
-            {
-                // Move the cursor to the current position
-                // in order to avoid a sudden jump when it
-                // becomes visible again
-                float scale = Width / (float)Size.Width;
-                SDL.WarpMouseInWindow(window.Handle,
-                    (int)Math.Round(MouseState.X / scale),
-                    (int)Math.Round(MouseState.Y / scale));
-            }
+        }
+
+        private void HideCursor(bool hide)
```

Comment:
Same as the other one, if it can both hide and unhide the mouse, split it into two methods without a parameter or rename the method.